src/TableFunctions/TableFunctionPostgreSQL.cpp

```diff
@@ -64,7 +65,8 @@ StoragePtr TableFunctionPostgreSQL::executeImpl(const ASTPtr & /*ast_function*/,
         String{},
         context,
         configuration->schema,
-        configuration->on_conflict);
+        configuration->on_conflict,
+        PostgreSQLSettings{});
```

Comment:
Let's support settings in `postrgesql` table function similar to `mysql` table function: https://github.com/ClickHouse/ClickHouse/blob/94264b11934f9515efbbf71d99c976fa60c51bab/src/TableFunctions/TableFunctionMySQL.cpp#L84-L93
e.g. it will work as `select * from postgresql(<host>, ..., SETTINGS setting1=value1, setting2=value2)`.